### ISSUE TITLE
Is some change needed in resnet50.py? Error while creating model

### ISSUE THREAD

**Issue by shirishr** [Arsey/keras-transfer-learning-for-oxford102#20]
I get error:
Negative dimension size caused by subtracting 3 from 2 for 'max_pooling2d_4/MaxPool' (op: 'MaxPool') with input shapes: [?,2,112,64].
I am using Python 3.5 and Tensorflow 1.2 keras 2.0.2

**Comment by shirishr** [comment]
I tried various combinations of values for:
image_dim_ordering and
image_data_format  the errors would be different but still related to dim mismatch.
So I abandoned Tensorflow and switched to Theano backend with

keras.backend.set_image_dim_ordering('tf')
keras.backend.set_image_data_format('channels_first')

This combination seems to work. The requirements.txt file in the Github specifically calls out theano (0.9.0) Hence closing the issue
